ova/generate_ova.sh

```diff
@@ -1,4 +1,5 @@
-#!/bin/bash
+#!/bin/sh
+set -exf
```

Comment:
This isn't really necessary.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/2629f18c2d161e7cfe70b71d39522548b5459f55